versioned_docs/version-latest/faq.md

```diff
@@ -44,7 +44,7 @@ https://docs.docker.com/desktop/
 <!-- #1221 -->
 #### **Q: Can I have Docker Desktop installed alongside Rancher Desktop?**
 
-**A:** Yes, but they cannot be run at the same time as both Rancher Desktop and Docker Desktop use the same Docker socket (`/var/run/docker.sock`). Be sure to stop one before starting the other.
+**A:** Yes, but they cannot be run at the same time as both Rancher Desktop and Docker Desktop use the same Docker socket (`/var/run/docker.sock`). Be sure to stop one before starting the other. (seems to be wrong)
```

Comment:
check https://github.com/rancher-sandbox/docs.rancherdesktop.io/pull/428#issue-3061485758 for details